src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -566,7 +566,7 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
         }
         
         adjustQuestionNumbers(oldQuestionNumber, newQuestionNumber, questions);
-        updateFeedbackQuestion(newQuestion);
+        updateFeedbackQuestionWithResponseRateCheck(newQuestion);
```

Comment:
Actually, I have a suspicion that the old <code>updateFeedbackQuestion</code> should not be used anywhere in the entire code at all. Can you check that for me? We can make a new issue and fix that there if so.